sdk-python/list_and_match_messages.py

```diff
@@ -0,0 +1,54 @@
+import argparse
+from datetime import datetime
+from awaazde.constants import CommonConstants, APIConstants
+from awaazde.utils import CSVUtils
+from awaazde import AwaazDeAPI
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(description='Check Created Messages')
+    parser.add_argument('username', type=str, help='Username of the tenant')
+    parser.add_argument('password', type=str, help='Password of the tenant')
+    parser.add_argument('organization', type=str, help='Organization of the tenant')
+    parser.add_argument('match_path', type=str, help='Path for the csv with which filtered data is to be matched')
+    parser.add_argument('--params', type=str,
+                        help='Fields on which filters are to be applied in the form: {"send_on__gt":"10-12-2020",'
+                             '"send_on__lt":"10-12-2020","tags":"dummy_tag"}')
```

Comment:
Is user actually providind this as a dictionary like this? Not very user friendly. Look for standard ways you can supply key-value pairs to a unix command